flight/Libraries/alarms.c

```diff
@@ -238,6 +238,150 @@ static int32_t hasSeverity(SystemAlarmsAlarmOptions severity)
     return 0;
 }
 
+static const char alarm_names[][8] = {
```

Comment:
This uses 8 bytes of flash for every alarm entry. I think you could conserve some space by using the following construct:

    static const char * alarm_names[] = {
        [SYSTEMALARMS_ALARM_OUTOFMEMORY] = "MEMORY",
        [SYSTEMALARMS_ALARM_CPUOVERLOAD] = "CPU",
        ...
    }